lib/s390base.pm

```diff
@@ -17,6 +22,13 @@ use utils;
 use strict;
 use warnings;
 
+=head2 copy_testsuite
+
+ copy_testsuite();
+
+Fetch the testcase and common libraries from openQA server
+
+=cut
 sub copy_testsuite {
     my ($self, $tc) = @_;
```

Comment:
There is a parameter here `$tc`, this needs to be mentioned in the documentation.